### DISCUSSION TITLE
ComblockEngine教程

### DISCUSSION THREAD

**Discussion by shangdibaozi** [kbengine/kbengine#1390] [General]
[KBEngineDoc.zip](https://github.com/kbengine/kbengine/files/6088335/KBEngineDoc.zip)
因为之前KBEngine官网停了好几次，然后就把教程文档都拉到本地。没想到这次论坛彻底关闭，现在后悔没把论坛的内容拉下来。

解压压缩包，双击里面的index.html就可以了。

**Comment by kebiao** [comment]
https://github.com/kbengine/kbengine/discussions/1367

以前论坛内容在这里

**Reply by shangdibaozi** [reply]
这个下载下来看过。阅读不是很方便。
